### PR TITLE
website/integrations: add outline knowledge base

### PR DESCRIPTION
**Pull request by fallenhitokiri**
## Details

Adds instructions to setup OIDC for outline.

---

## Checklist

If applicable

-   [x] The documentation has been updated
-   [x] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/integrations/services/outline/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+> -- https://www.getoutline.com
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `outline.company` is the FQDN of the Outline install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+Create an OAuth2/OpenID provider with the following parameters:
```
Comment (tanberry) [inline website/integrations/services/outline/index.md:21]:
```suggestion

## authentik configuration

1. Create an OAuth2/OpenID provider with the following parameters:
```

File: website/integrations/services/outline/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+-   Client Type: `Confidential`
+-   Scopes: OpenID, Email and Profile
+-   Signing Key: Select any available key
+-   Redirect URIs: `https://outline.company/auth/oidc.callback`
+
+Note the Client ID and Client Secret values.
```
Comment (tanberry) [inline website/integrations/services/outline/index.md:28]:
```suggestion
2. Note the Client ID and Client Secret values.
```

File: website/integrations/services/outline/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Outline
```
Comment (tanberry) [inline website/integrations/services/outline/index.md:30]:
```suggestion
## Outline configuration
```

File: website/integrations/services/outline/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+You need to set the following `env` Variables for Docker based installations.
```
Comment (tanberry) [inline website/integrations/services/outline/index.md:32]:
```suggestion
You need to set the following `env` variables for Docker-based installations.
```

File: website/integrations/services/outline/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Set the following values:
```
Comment (tanberry) [inline website/integrations/services/outline/index.md:34]:
```suggestion
1. Set the following values:
```

**Comment by tanberry** [comment]
Thanks so much @fallenhitokiri for this contribution! I made a few suggestions/edits, and also you will need to edit the `sidebarintegrations.js` file to add Outline to the left nav pane, please. Let us know if you have any questions; we look forward to merging this as soon as those few fixes are made. Thanks again!

**Review by tanberry — CHANGES_REQUESTED** [review]

**Comment by tanberry** [comment]
Hi again @fallenhitokiri will you have a chance to make the requested changes, and then we can merge it? Or, if you're ok with it, I can make the changes and merge it.

**Comment by fallenhitokiri** [comment]
@tanberry I got started on the changes and expect to land them tomorrow or Tuesday. (I didn’t get a notification and only saw your comments by accident when checking my GitHub inbox)

I used the HedgeDoc documentation as a template. Would some of the changes suggested here also apply to the HedgeDoc doc?

**Comment by tanberry** [comment]
Hi @fallenhitokiri thanks for the update, and I will take a look at the HedgeDoc docs as well. We do have a proper template there in the repo, but we need to do a better job of making it easier to find, for sure.

We will look forward to your changes; let me know if we can help!

**Review by tanberry — APPROVED** [review]
Thanks @fallenhitokiri !!

**Pull request merged**